docs/content/latest/integrations/kairosdb.md

```diff
@@ -73,7 +73,7 @@ Use HELP for help.
 ycqlsh>
 ```
 
-- Run a cql command to verify it is working:
+- Run the following YCQL commands to verify it is working:
 
 ```sql
 ycqlsh> describe keyspaces;
```

Comment:
Our docs are so inconsistent with uppercase vs lowercase statements. And I know the interpreters are case-insensitive, but it's definitely something of a religious war on the order of vi vs emacs. Bryn hates uppercase, so I suppose that's as good a reason as any to standardize on lowercase. ;-)

(I think the main argument for uppercase is making the statements stand out, but isn't that what syntax highlighting is for in modern docs?)